nevergrad/optimization/test_callbacks.py

```diff
@@ -112,3 +114,33 @@ def test_early_stopping() -> None:
     # below functions are inlcuded in the docstring
     assert optimizer.current_bests["minimum"].mean < 12
     assert optimizer.recommend().loss < 12  # type: ignore
+
+
+def test_optimization_logger(caplog) -> None:
+    with caplog.at_level(logging.INFO):
```

Comment:
can you ut this just before the call to minimize? Let's keep context as small as possible (this way it's clearer what it captures)
I'm actually wondering if the context could be around *only* the minimize method, I think you can check the text afterwards even if you are no more in the context, so that would be even better.

Reply:
I've implemented these recommendations and you are correct. The context is now only around the minimize method with the assertions outside of the context.